src/test/java/bio/terra/integration/SamFixtures.java

```diff
@@ -91,6 +92,43 @@ public void deleteSnapshotAccessRequest(
     }
   }
 
+  public void addGroup(TestConfiguration.User user, String groupName) {
+    try {
+      HttpHeaders authedHeader = getHeaders(user);
+      String accessToken = getAccessToken(authedHeader);
+      GroupApi samGroupApi = new GroupApi(getApiClient(accessToken));
+      // what is supposed to be in the body?
+      samGroupApi.postGroup(groupName, null);
```

Comment:
I think if we're just adding a group, nothing needs to be added? It seems like:
```
samGroupApi.postGroup(groupName, null) 
```
is called in several places